demo-app/src/components/aruba/ArubaMap.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { Box } from 'grommet';
+import { Grommet } from 'grommet-icons';
+import {
+  Cluster,
+  Controls,
+  Map,
+  Marker,
+  MarkerCluster,
+  Pin,
+} from 'grommet-leaflet-core';
+
+import { userLocation, formatLocationsToLatLng } from '../../utils/locations';
+import { generateArubaLocations } from './ArubaData';
+import { ClusterPopup } from '../../ClusterPopup';
+import { ArubaPin } from './ArubaPin';
+import { ArubaPopup } from './ArubaPopup';
+import { hpeLeaflet } from '../../themes';
+import { getClusterStatus, getClusterSize } from '../../utils/status';
+
+function ArubaMap() {
+  const [geolocation, setGeolocation] = useState();
+  const [locations, setLocations] = useState();
+  const containerRef = useRef();
+  const mapContainerRef = useRef();
+
+  // get the user's location
+  useEffect(() => {
+    userLocation().then(location => {
+      setGeolocation(location);
+      setLocations(generateArubaLocations(20, { center: location, radius: 6 }));
+    });
+  }, []);
+
+  return (
+    <Box ref={containerRef} flex background="background-contrast">
+      {geolocation && (
+        <Map
+          id="map"
+          ref={mapContainerRef}
+          center={geolocation}
+          zoom={6}
+          zoomControl={false}
+          theme={hpeLeaflet}
+        >
+          <Controls locations={formatLocationsToLatLng(locations)} />
+          <Marker position={geolocation} icon={<Grommet />} />
+          <MarkerCluster
+            popup={cluster => <ClusterPopup cluster={cluster} />}
+            icon={cluster => {
+              const kind = getClusterStatus(cluster.getAllChildMarkers());
+              const size = getClusterSize(cluster);
```

Comment:
From a visual standpoint, the clusters are the same size as the pins which makes it difficult to scan. I'm wondering if we should use the "medium" cluster in this case so it's bigger than the individual pins?

Reply:
Changed to 'medium'